ui/tests/acceptance/enterprise-replication-test.js

```diff
@@ -88,6 +89,22 @@ module('Acceptance | Enterprise | replication', function(hooks) {
     await click('[data-test-secondary-add]');
 
     await pollCluster(this.owner);
+    // checks on secondary token modal
+    assert.dom('#modal-wormhole').exists();
+
+    const date = new Date();
+    date.setMinutes(date.getMinutes() + 30); // add default 30 min TTL to current date to return expires
+    const dateFromHelper = dateFormat([date, 'MMM DD, YYYY hh:mm:ss A']);
```

Comment:
this could be renamed `formattedDate`. `dateFromHelper` makes me think that it's returning the actual helper, vs. a string

Reply:
I added the chain the assert.equal as I was having issues chaining on dateFormat and document.query...